test/bats/aws.bats

```diff
@@ -5,11 +5,16 @@ load helpers
 WAIT_TIME=120
 SLEEP_TIME=1
 PROVIDER_YAML=https://raw.githubusercontent.com/aws/secrets-store-csi-driver-provider-aws/main/deployment/aws-provider-installer.yaml
-NAMESPACE=kube-system
+export NAMESPACE="sscsi-namespace"
```

Comment:
In a general scenario, the application pod doesn't get deployed in the `kube-system` namespace.